.github/workflows/main.yml

```diff
@@ -60,7 +60,7 @@ jobs:
       run: |
         mix deps.get --check-locked
         cd test_integrations/phoenix_app
-        mix deps.get --check-locked
+        mix deps.get
```

Comment:
FYI this would not pass under 1.16 for some reason. Lock is different under this version of Elixir :/